suite-native/device/src/hooks/useDelayedNavigation.ts

```diff
@@ -41,8 +49,14 @@ export const useDelayedNavigation = () => {
     }, [navigation]);
 
     // If Device is authorized and loading accounts, redirect to the Home screen.
+    // If connected device has no supported networks and coin enabling init was finished,
+    // also redirect to the Home screen otherwise user would be blocked forever.
+    // This can happen if user enabled only COIN on device A and then connects device B which does not support COIN.
     useEffect(() => {
-        if (isDeviceReadyToUseAndAuthorized && isTimeoutFinished) {
+        if (
+            (isDeviceReadyToUseAndAuthorized && isTimeoutFinished) ||
+            (deviceEnabledDiscoveryNetworkSymbols.length === 0 && isCoinEnablingInitFinished)
```

Comment:
should not be coin enabling screen showed instead if user has no coins enabled?

Reply:
Because this will show empty dashboard where is prompt to add new account and this will also enable the network.

Reply:
I've moved this hook in 75a57dc15c to the package to where it is used to solve circular dependency